app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+  has_secure_password
+  validates :fname, :lname, :email, presence: true
```

Comment:
@jynti Please add the following validations:

1. fname, lname should not be of more than 255 characters length.
2. email should be unique.
3. password and password_confirmation presence. Ignore if already present.